internal/advertisement-operator/remote-watcher.go

```diff
@@ -3,46 +3,63 @@ package advertisementOperator
 import (
 	discoveryv1alpha1 "github.com/liqotech/liqo/api/discovery/v1alpha1"
 	advtypes "github.com/liqotech/liqo/api/sharing/v1alpha1"
+	"github.com/liqotech/liqo/pkg/crdClient"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/tools/cache"
 	"k8s.io/klog"
+	"os"
+	"time"
 )
 
 func (b *AdvertisementBroadcaster) WatchAdvertisement(homeAdvName string) {
+	var err error
+	resyncPeriod := 1 * time.Second
```

Comment:
Do we need a so short resync period on the remote cluster?

Reply:
I don't think so, I didn't think about it.. thanks!

Reply:
actually I think we could also set it to 0, so that it is only triggered by Watch event and we don't annoy the remote cluster with periodic List (since this watcher should be triggered very few times). what do you think @palexster?

Reply:
My 2 cents: I would force a resync in order to recover from deadlocks or missed events ( If I understand the scenario correctly). For example, 60 seconds should be ok.